pcdsdevices/ipm.py

```diff
@@ -443,9 +461,9 @@ class IPM_Det(BaseInterface, Device):
     """Base class for IPM_IPIMB and IPM_Wave8. Not meant to be instantiated."""
     tab_component_names = True
 
-    def isum(self):
-        """Returns the detector's isum value."""
-        return self.det.isum.get()
+    def sum(self):
```

Comment:
Seems a bit odd to have accessor methods like these which alias component names of `det`, but it appears to be a common feature here, so 🤷‍♂️

Reply:
I largely agree that this pattern can be very confusing, as helpful as it would be sometimes.